modules/core/src/encoder/field/PurposeRestrictionVectorEncoder.ts

```diff
@@ -21,57 +21,74 @@ export class PurposeRestrictionVectorEncoder {
         bitString += IntEncoder.encode(purpRestriction.purposeId, BitLength.purposeId);
         bitString += IntEncoder.encode(purpRestriction.restrictionType, BitLength.restrictionType);
 
-        // now get all the vendors under that restriction
+        // Get available vendor IDs from the GVL
+        const gvlVendors = [...prVector.gvl.vendorIds];
```

Comment:
Why destructure a `Set` (O(1)) look up in favor of an `Array`?
https://github.com/InteractiveAdvertisingBureau/iabtcf-es/blob/353e0e349544794b6523ae04238dfbc20a6f94ba/modules/core/src/GVL.ts#L192